src/en/overlay/pijuice-hat.md

```diff
@@ -40,15 +42,33 @@ PiJuice is a fully uninterruptable / uninterupted power supply that will always
 
 The tri-coloured RGB LEDs will allow you to keep track of the battery charge levels and other info (they are programmable). There are also three programmable buttons which will allow you to trigger events or customisable scripts aside from their predefined functions. PiJuice only uses five of the Pi's GPIO pins (just power and I2C), the rest are free and made available via the stacking header which allows for other boards to be used along side PiJuice. The board can be powered directly from the grid with the standard Raspberry Pi PSU, via an on board battery, via external batteries, solar panels, wind turbines and other renewable sources.
 
-The board has a Raspberry Pi HAT compatible layout, with onboard EEPROM (you can disable the EEPROM if you want also). It has been designed for the Raspberry Pi A+, B+, 2B, 3B and 3B+ but it is also electrically compatible with the Raspberry Pi Zero v1.3 and Zero W v1.1 or any other Pi.
+The board has a Raspberry Pi HAT compatible layout, with onboard EEPROM (you can disable the EEPROM if you want also). It has been designed for the Raspberry Pi A+, B+, 2B, 3B, 3B+ and 4B, Raspberry Pi Zero v1.3, Raspberry Pi Zero Wireless, and any other Raspberry Pi with 40 GPIO pins.
```

Comment:
Seems a bit of a stretch to claim that it "has been designed for the Raspberry Pi 4B", when the PiJuice HAT was released before the Pi4 was released?

Reply:
This is probably something that more widely affects their own product nomenclature- if the designers are happy to stand behind it being electrically compatible without faults or weirdness then I don't see any harm in this clairvoyant wording. From experience, it's a dangerous claim to make without testing first. I have at least one product in my care that behaves differently on the Pi 3B+ versus the Pi 4 due to some elusive electrical wierdness or initial pin state change.

Reply:
What do you recomends?, something like:
`It has been designed for the Raspberry Pi A+, B+, 2B, 3B and 3B+ but it is also electrically compatible with the Raspberry Pi 4B, Zero v1.3, Raspberry Pi Zero Wireless, and any other Raspberry Pi with 40 GPIO pins.`

Reply:
This is up to @Gadgetoid (as it's his site) and/or @PiSupply (as it's their product), but as an "outside observer" I personally don't see what's wrong with "It has been designed for all models of Raspberry Pi with a 40-pin GPIO header." :man_shrugging: (which is basically the purpose of the HAT spec)

If it's compatible with everything, is there really a need to explicitly mention every model of Raspberry Pi?

Reply:
Happy to have the suggested text. It was not meant as "compatible with" rather then designed for. Sorry for any confusion.

Reply:
I'd merged the suggested text in the new commit.